types/gestalt/index.d.ts

```diff
@@ -197,35 +215,39 @@ export interface ButtonProps {
         | React.KeyboardEvent<HTMLAnchorElement>
         | React.KeyboardEvent<HTMLButtonElement>
     >;
-    ref?: React.Ref<'button'> | React.Ref<'a'>;
```

Comment:
how does this affect https://gestalt.netlify.app/Button#ref ? Seems like passing `ref` to button is a documented use-case